pci-dss/require-network-policy.yaml

```diff
@@ -0,0 +1,389 @@
+apiVersion: kyverno.io/v1
```

Comment:
This policy is unusually long. Seems repetitive. Please check.

Reply:
Done! Please check..

Reply:
@dolisss Looks like this file has some copy-paste error. It's malformed. This policy `require-network-policy` is repeated about 7-8 times.

Reply:
@anusha94 I have updated the policy, Please take a look